ultimarc/schemas/ultrastik-config.schema

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": [
+    "schemaVersion",
+    "resourceType",
+    "deviceClass",
+    "mapSize",
+    "restrictor",
+    "flash",
+    "borders",
+    "map"
+  ],
+  "properties": {
+    "schemaVersion": {
+      "$id": "#root/schemaVersion",
+      "title": "Schemaversion",
+      "type": "integer",
+      "const": 2.0
+    },
+    "resourceType": {
+      "$id": "#root/resourceType",
+      "title": "Resourcetype",
+      "type": "string",
+      "const": "ultrastik-config"
+    },
+    "deviceClass": {
+      "$id": "#root/deviceClass",
+      "title": "Deviceclass",
+      "type": "string",
+      "const": "ultrastik"
+    },
+    "mapSize": {
+      "$id": "#root/mapSize",
+      "type": "number",
+      "const": 9
```

Comment:
Thinking that since this is a const number to take it out of the config.  Thoughts?

Reply:
I'm Ok with taking it out.